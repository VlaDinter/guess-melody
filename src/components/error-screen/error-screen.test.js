import React from 'react';
import renderer from 'react-test-renderer';

import ErrorScreen from './error-screen.jsx';

it(`ErrorScreen is rendered correctly`, () => {
  const tree = renderer.create(<ErrorScreen/>).toJSON();

  expect(tree).toMatchSnapshot();
});
