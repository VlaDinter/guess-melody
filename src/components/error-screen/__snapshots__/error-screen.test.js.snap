// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen is rendered correctly 1`] = `
<section
  className="modal"
>
  <h2
    className="modal__title"
  >
    Произошла ошибка!
  </h2>
  <p
    className="modal__text"
  >
    Статус: 404. Пожалуйста, перезагрузите страницу.
  </p>
</section>
`;
